[INTERNAL FIX] installer-about-ip-nits (#65304)
openshift/openshift-docs: modules/installation-overview.adoc

[id="installation-overview_{context}"]
= About {product-title} installation

The {product-title} installation program offers four methods for deploying a cluster. These methods are detailed in the following list:

* *Interactive*: You can deploy a cluster with the web-based link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[{ai-full}]. This is an ideal approach for clusters with networks connected to the internet. The {ai-full} is the easiest way to install {product-title}, it provides smart defaults, and it performs pre-flight validations before installing the cluster. It also provides a RESTful API for automation and advanced configuration scenarios.

> * *Local Agent-based*: You can deploy a cluster locally with the agent-based installer for air-gapped or restricted networks. It provides many of the benefits of the {ai-full}, but you must download and configure the link:https://console.redhat.com/openshift/install/metal/agent-based[agent-based installer] first. Configuration is done with a command-line interface. This approach is ideal for air-gapped or restricted networks.

Author: Nice suggestions. I've seen these terms used in some of our published docs, so a further update might be needed. I do not think my PR can cover does as Nutanix docs use "ait-gapped" frequently and maybe this is a well-known term in that area. Such updates to a doc set would need specific SME reviews.

> * *Local Agent-based*: You can deploy a cluster locally with the Agent-based Installer for disconnected environments or restricted networks. It provides many of the benefits of the {ai-full}, but you must download and configure the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. Configuration is done with a command-line interface. This approach is ideal for disconnected environments or restricted networks.

Reviewer: ```suggestion
* *Local Agent-based*: You can deploy a cluster locally with the Agent-based Installer for disconnected environments. It provides many of the benefits of the {ai-full}, but you must download and configure the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. Configuration is done with a command-line interface. This approach is ideal for disconnected environments.
```
Same comment here about redundancy

* *Automated*: You can deploy a cluster on installer-provisioned infrastructure. The installation program uses each cluster host's baseboard management controller (BMC) for provisioning. You can deploy clusters on dual-connected, air-gapped, restricted networks.

* *Full control*: You can deploy a cluster on infrastructure that you prepare and maintain, which provides maximum customizability. You can deploy clusters on dual-connected, air-gapped, restricted networks.

> Each method shares the following characteristic for deploying a cluster:

Reviewer 2: Granted, I'm not sure what this paragraph and following list is trying to achieve, since the list has more to do with OCP cluster characteristics than installs, but here's a rewrite that I think improves the original wording while keeping its meaning:

```suggestion
Each method deploys a cluster with the following characteristics:
```

> * Highly available infrastructure with no single points of failure is available by default.

Reviewer 2: Out of scope, but I don't think this line reads as a "phrase that represents a singular characteristic". Could you consider the following?

```suggestion
* Highly available infrastructure with no single points of failure, which is available by default.
```

* Administrators can control what updates are applied and when.

[id="about-the-installation-program"]
== About the installation program

> You can use the installation program to deploy each type of cluster. The installation program generates the main assets, such as Ignition config files for the bootstrap, control plane (master), and worker machines. You can start an {product-title} cluster with these three machine configurations, provided you correctly configured the infrastructure.

Reviewer: ```suggestion
You can use the installation program to deploy each type of cluster. The installation program generates the main assets, such as Ignition config files for the bootstrap, control plane, and compute machines. You can start an {product-title} cluster with these three machine configurations, provided you correctly configured the infrastructure.
```
Removing master/worker per conscious language guidelines.

Author: Thanks. Updated. 

I was unsure on this one as I still see "master" and "worker" dotted around in the docs. I know changing codebase names is out of our control, but general references are. Perhaps, I could update the complete install docs to replace the terms. WDYT?

Reviewer: If you've got a ton of spare time, feel free! it would be one of those huge PRs where you post it in team-docs and a dozen people look over it
